packages/components/pie-button/README.md

```diff
@@ -1,38 +1,145 @@
+| `formenctype`    | `"application/x-www-form-urlencoded"`, `"multipart/form-data"`, `"text/plain"`                                                                                           | If the button is a submit button, specifies how to encode the form data that is submitted.                                                                                                                          | `undefined`    |
+| `formmethod`     | `"post"`, `"get"`, `"dialog"`                                                                                                                                            | If the button is a submit button, this attribute specifies the HTTP method used to submit the form.                                                                                                                 | `undefined`    |
+| `formnovalidate` | `true`, `false`                                                                                                                                                          | If the button is a submit button, this Boolean attribute specifies that the form is not to be validated when it is submitted.                                                                                       | `undefined`    |
+| `formtarget`     | `"_self"`, `"_blank"`, `"_parent"`, `"_top"`                                                                                                                             | If the button is a submit button, this attribute is an author-defined name or standardized, underscore-prefixed keyword indicating where to display the response from submitting the form.                          | `undefined`    |
+
+### Slots
+
+| Slot     | Description                                                                                                                                                                                                                                                                         |
+|----------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `default`| The default slot is used to pass text into the button component.                                                                                                                                                                                                                    |
+| `icon`   | Used to pass in an icon to the button component. The icon placement can be controlled via the `iconPlacement` prop and we recommend using `pie-icons-webc` for defining this icon, but this can also accept an SVG icon.                                                           |
+
+### CSS Variables
```

Comment:
Do you think we should omit the section so we don't have information overload for users?

Reply:
I don't, personally. I think the structure is important and this makes it extremely clear rather than them looking elsewhere in case they are just undocumented